static/app/components/events/breadcrumbs/breadcrumbsDataSection.tsx

```diff
@@ -194,6 +199,35 @@ export default function BreadcrumbsDataSection({
   );
 }
 
+function BreadcrumbsFeedback() {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const feedback = useFeedbackWidget({
```

Comment:
@leeandher I actually just added a different way of doing this if you're interested. You can just use `useFeedbackForm()` which will return an `openForm()` function

Reply:
> To allow feedback to be submitted properly, I had to prevent outside clicks from closing the drawer.

You won't have to do this actually with `useFeedbackForm()`! I added it because I had the same problem with the search bar.

Reply:
I implemented the new hook (thanks!) but I still had the same problem with outside clicks. The drawer is set to close if a click is registered on any element that isn't a child of the drawer, so since the form is rendered outside it, it still has the same issue. Is there something I missed?

Reply:
Ah yes you're right that it doesn't prevent that. The thing useFeedbackForm avoids is the feedback form being closed when the trigger unmounts.

Something you may want to look at is using `useOverlayAria()` (which we use in useOverlay) for the drawer component. That accepts a `shouldCloseOnInteractOutside` prop which you could use here to prevent feedback interactions from closing the drawer.